test/validator/samples/a11y-aria-proptypes-token/input.svelte

```diff
@@ -1,5 +1,5 @@
 <div aria-sort="" />
-<div aria-sort="descnding" />
+<div aria-sort="incorrect" />
```

Comment:
Ok, this item is intentionally not a correct token. Instead of using a misspelled word (which is confusing), I've selected `incorrect` which I hope satisfies the spirit of the parsing complaint while retaining its length.